simplq/src/components/pages/Admin/DeleteQueue.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import DeleteIcon from '@material-ui/icons/Delete';
+import styles from '../../../styles/adminPage.module.scss';
+
+export default () => {
+  //   const [open, setOpen] = React.useState(false);
```

Comment:
We can remove from here and below occurances?

Reply:
Yeah